test/controllers/geocoder_controller_test.rb

```diff
@@ -57,6 +88,7 @@ def test_identify_latlon_ne_d
       "50.06773N, 14.37742E"
     ].each do |code|
       latlon_check code, 50.06773, 14.37742
+      assert_nil @controller.params[:latlon_digits]
```

Comment:
Why do we want to assert this? The only thing this test should be concerned about is what results it gets from the controller method - how the controller does that internally is none of it's concern?

Reply:
This entire file tests `@controller.params` all over the place. It has been doing it for a long time (https://github.com/openstreetmap/openstreetmap-website/commit/aaeca5b5346b01d7ef26a301492804d36049eff8) and you didn't mind adding more recently (https://github.com/openstreetmap/openstreetmap-website/commit/6d217da3b65be4f2973c536fd8fb0765e2a6e450).